.gitignore

```diff
@@ -36,6 +36,9 @@ proguard/
 /.idea/gradle.xml
 /.idea/libraries
 *.iml
+.idea/inspectionProfiles/Project_Default.xml
```

Comment:
+1.  And thanks for killing off those other ignored files so the changes stop being tracked!